carbon-design-system/carbon
# fix: arrow breaking in AILabel popover

Closes #17965 

Arrow looks breaking in the AI explainability popover in case of small icons.

#### Changelog

added alignmentAxis in offset for popover.

**Changed**

passing a prop from AILabel to Toggletip component in case of 'XS', '2xs' or 'mini' icon.

#### Testing / Reviewing

1. Go to Storybook > AILabel > Default
2. hover on AI Label, the popover should not looks breaking.
